package.json

```diff
@@ -16,21 +16,25 @@
     "make-docs": "rm -rf api_docs && typedoc --plugin typedoc-plugin-markdown --options typedoc.config.js .",
     "metadata-docs": "node ./scripts/MetadataMdWrapper.js",
     "generate-slim-metadata": "node ./scripts/generateSlimMetadataWrapper.js",
-    "lint": "npx tslint -c tslint.json --project . && npx prettier --list-different './packages/**/src/*.ts'"
+    "lint": "eslint -c .eslintrc.js --ext .ts"
```

Comment:
should this still be `npx eslint` otherwise it requires global install?

Reply:
Ok it works using local deps.
turns out PATH will be extended to include: `./node_modules/.bin/eslint`